espwrap/base.py

```diff
@@ -58,99 +58,52 @@ def add_recipient(self, email, name='', merge_vars={}):
         # tuple for performance
         self.recipients = itertools.chain(self.recipients, (recip,))
 
-        return self
-
-    def add_recipients(self, recipients):
+    def add_recipients(self, *recipients):
         for recip in recipients:
             self.add_recipient(recip)
 
-        return self
-
     def clear_recipients(self):
         self.recipients = list()
 
-        return self
-
-    def add_global_merge_var(self, key, value, overwrite=False):
-        if not overwrite and self.global_merge_vars.get(key):
-            raise Exception(
-                'Global merge var {} has already been set'.format(key)
-            )
-
-        self.global_merge_vars[key] = value
-
-        return self
-
-    def add_global_merge_vars(self, mvars):
-        for var in mvars:
-            self.add_global_merge_var(**var)
-
-        return self
+    def add_global_merge_vars(self, **kwargs):
+        for key, val in kwargs:
```

Comment:
I'll change that to `.items()`, I completely forgot this (hence why I need unit tests....)

`.iteritems()` is gone in Py3 and I explicitly want this lib to be as polyglot as possible, given that it's 2016 and Py2 should be long dead by now.